Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/DataBindingAdapters.kt

```diff
@@ -29,7 +33,7 @@ fun setAnimation(view: LottieAnimationView, animation: Int?) {
         val type = appContext.resources.getResourceTypeName(animation)
 
         if (type == DRAWABLE_TYPE) {
-            view.setImageDrawable(appContext.getDrawable(animation))
+            view.setImageDrawable(appContext.getDrawableCompat(animation))
```

Comment:
here you could use the context of the view itself

Reply:
Good point, I think I changed this in one of my screenshots PRs. I will check it there again as there are unit tests covering such change which I already modified #2042

Reply:
this commit addressed the change .https://github.com/corona-warn-app/cwa-app-android/pull/2042/commits/6cbd6b1abca296fb5e70252b597094364ed6d513